Fixed arrangement of Search and New buttons

peach280 opened the pull request:
# Pull Requests Requirements

- [X] The PR title includes a brief description of the work done, including the
      Issue number if applicable.
- [X] The PR includes a video showing the changes for the work done.
- [X] The PR title follows conventional commit label standards.
- [X] The changes confirm to the OpenElis Global x3 Styleguide and design
      documentation.
- [X] The changes include tests or are validated by existing tests.
- [X] I have read and agree to the Contributing Guidelines of this project.

## Summary
Fixes #1890 

## Screenshots

![Screenshot 2025-04-10 150151](https://github.com/user-attachments/assets/94e72680-2f16-4ca3-9124-aa3ba2e79e21)

peach280 committed:
Fixed arrangement of Search and New buttons

Signed-off-by: Vaishnavi Bhandari <vaishnavibhandari.128@gmail.com>

mozzy11 commented:
Thanks @peach280 . 
But the issue is by default we dont add a size to most of the buttons. 
So we let Carbon use the default button size

peach280 committed:
Removed the size property from buttons

Signed-off-by: Vaishnavi Bhandari <vaishnavibhandari.128@gmail.com>

peach280 committed:
Merge branch 'develop' into buttons

peach280 commented:
Hi @mozzy11 I removed the size property from it and adjusted the layout without size. Here is the result

![Screenshot 2025-04-11 132516](https://github.com/user-attachments/assets/77d7d36f-a7a2-4861-bb42-6ac20ff8a794)

mozzy11 commented:
Thanks @peach280 . Can you also attach the desktop view before and after to ensure its consistent ??

peach280 commented:
Here are the screenshots for desktop view

Before:

![Screenshot 2025-04-15 191909](https://github.com/user-attachments/assets/730e92c3-f18e-4d17-836b-103cbc70f661)

After:

![Screenshot 2025-04-15 191927](https://github.com/user-attachments/assets/97355a12-ab0b-4b87-b0b2-fc3b532cb304)

peach280 committed:
Merge branch 'develop' into buttons

peach280 committed:
Merge branch 'develop' into buttons

peach280 committed:
Merge branch 'develop' into buttons

peach280 committed:
Merge branch 'develop' into buttons

peach280 committed:
Merge branch 'develop' into buttons

pmanko commented:
Hi @peach280,

We're closing this PR due to extended inactivity and duplication.

**Summary:**
- Created: April 10, 2025 (9 months ago)
- Last activity: May 6, 2025 (9 months ago)
- Current status: Mergeable, but duplicate of #1960

This PR addresses the same issue (#1890) as PR #1960, which was recently closed due to inactivity. This PR has also been inactive for 9 months with no reviews or activity.

**Next Steps:**
If you'd like to revisit this work, please feel free to:
- Rebase against the latest `develop` branch
- Reopen this PR or create a new one

Thank you for your contribution! We appreciate your effort and would welcome future submissions.

---
**Backlog Cleanup Initiative** - Closing stale PRs to improve project visibility. See [PR Backlog Analysis](https://uwdigi.atlassian.net/wiki/spaces/OI/pages/1023705105/PR+Backlog+Analysis).
